src/PanelTraits/Query.php

```diff
@@ -53,6 +53,23 @@ public function orderBy($field, $order = 'asc')
         return $this->query->orderBy($field, $order);
     }
 
+    /**
+     * Order the results of the query in a custom way.
+     */
+    public function customOrderBy($column, $column_direction)
+    {
+        if (isset($column['orderLogic'])) {
+            $this->query->getQuery()->orders = null;
+        
+            $orderLogic = $column['orderLogic'];
+
+            if (is_callable($orderLogic)) {
```

Comment:
What if it's not callable?

Reply:
Well, I just returned the ```$this->query``` as a fallback. At first I wanted to add a case for a string such as a different column name from the model, but I don't think that's a use case (clicking the sort link on some column and actually returning a sorting by another attribute in the same model).

The idea as I see it is to add functionality for sorting by a field from some other related model.

BTW, thanks for all your awesome work on 3.3! this package is a life saver 👍